src/App.tsx

```diff
@@ -1,26 +1,83 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
-import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import { UserWarning } from './UserWarning';
+import { USER_ID } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm';
+import { Footer } from './components/Footer';
+import { countActiveTodos } from './utils/countActiveTodos';
```

Comment:
As for me, I would create an 'index.ts' in _utils, components and etc directories_ and performed only one import in this component. Like:
`import { TodoList, TodoForm, Footer, Error, TodoInfo } from './components';`

This makes your code more readable and understandable.